Tests/OktaOAuth2Tests/DeviceAuthorizationFlowErrorTests.swift

```diff
@@ -0,0 +1,134 @@
+//
+// Copyright (c) 2021-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+// The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+//
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and limitations under the License.
+//
+
+import XCTest
+@testable import TestCommon
+@testable import AuthFoundation
+@testable import OktaOAuth2
+
+final class DeviceAuthorizationFlowErrorTests: XCTestCase {
+    let issuer = URL(string: "https://example.com")!
+    let clientMock = OAuth2ClientMock()
+    let urlSession = URLSessionMock()
+    var client: OAuth2Client!
+    var flow: DeviceAuthorizationFlow!
+
+    override func setUpWithError() throws {
+        client = OAuth2Client(baseURL: issuer,
+                              clientId: "clientId",
+                              scopes: "openid profile",
+                              session: urlSession)
+        JWK.validator = MockJWKValidator()
+        Token.idTokenValidator = MockIDTokenValidator()
+        Token.accessTokenValidator = MockTokenHashValidator()
+        flow = client.deviceAuthorizationFlow()
+    }
+    
+    override func tearDownWithError() throws {
+        JWK.resetToDefault()
+        Token.resetToDefault()
+    }
+    
+    func testSlowDown() throws {
```

Comment:
The down side I see to these tests are we aren't asserting anything about what delay is happening.

Here's the related test in okta-mobile-kotlin, that asserts the timeouts given https://github.com/okta/okta-mobile-kotlin/pull/186/files#diff-ee0407074640dcfed6eb5fa913fb504cbee134b4bdf152f89b9e11efe3175091R285

Reply:
Yes, I saw how you did that in your tests. I don't have an easy way of tracking that in my tests but ... wait, never mind, I just thought of a way that might work. 😄  Thanks for letting me rubber-duck on this.